blazor/datagrid/columns.md

```diff
@@ -511,6 +511,69 @@ In the following code sample, you can prevent default filter query generation us
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-prevent-query-generation-for-foriegnkey-column).
 
+### Enable multiple foreign key columns
+
+Multiple foreign key columns can be enabled by defining the multiple `GridForeignColumn` directives.
```

Comment:
Explanation is very simple. As i already informed like do the document is the following flow
1. What is the use of this requirement in usecase wise as as general requirement
2. how to achieve this using grid properties
3. And how we achieved in the below example

Reply:
Content edited.